modules/vpc/versions.tf

```diff
@@ -17,6 +17,6 @@
 terraform {
   required_version = "~> 0.12.6"
   required_providers {
-    google = "<4.0,>= 2.12"
+    google = "<4.0,>= 2.4"
```

Comment:
I see that `google_compute_network` added `delete_default_routes_on_create` with `2.4.0` but just wondering what was the reason for this since we already require a minimum of version of `2.12.0`.

Reply:
Mistake on my part. Thanks for catching that!